neurogym/wrappers/monitor.py

```diff
@@ -461,3 +496,120 @@ def plot_training_history(
             logger.info(f"Figure saved to {save_path}")
 
         return fig
+
+    def record_activations(
+        self,
+        layer: nn.Module,
+        name: str | None = None,
+        steps: int | None = None,
+    ) -> ActivationMonitor:
+        """Record the output activations of a layer over a trial.
+
+        Args:
+            layer: The layer whose activations are being monitored.
+            name: The name to use for the activation monitor.
+                This can be useful for retrieving activation monitors
+                at a later stage.
+            steps: The steps to record for.
+                This could be less than the total number of steps in a trial.
+
+        Returns:
+            An ActivationMonitor instance.
+        """
+        # Ensure that we have a name for this monitor
+        if name is None:
+            name = layer.__class__.__name__
+
+        # The total number of steps to record during each trial.
+        if steps is None:
+            steps = int(self.tmax / self.dt)
+
+        # Create an activation monitor
+        am = ActivationMonitor(layer, steps, name)
+
+        # Check if we are replacing an existing monitor
+        if name in self.activations:
+            logger.warning(f"Replacing an existing activation monitor '{name}'.")
```

Comment:
what is this warning for? I'm not sure the info given will be useful to users (i at least don't really know what to make of this information).